### ISSUE TITLE
[Bug]: dd does not return any output in the version

### ISSUE THREAD

**Issue by ismailnakkar** [Roboroads/laravel-tinker#232]
### Short bug description

dd does not return any output in the version

### Your project is not the problem

- [X] I tried using a fresh laravel install - I'm fairly certain it has nothing to do with my project.

### How to reproduce?

1- use dd in your project
2- use the plugin
3- the content of what should dd return does not appear

### Expected behavior

dd content should appear (like previous versions of the plugin)

### IDE & Version

PHPstorm 2023.3.1

### PHP Version

8.1

### OS and version

Linux

### Laravel Tinker plugin version

2.5.1

### Interpreter type

Local

### Laravel Version

9.45.1

### Tinker version

2.7.3

### Psysh version

0.11.10

**Comment by Roboroads** [comment]
Yes! And I actually already know why. I try to get just the output from Tinker/Psysh by wrapping the oputput in tokens (%START%tinkeroutput%END%). When you use dd(), execution gets terminated before %END% is displayed. 

I can fix this one, thanks for letting me know!

**Comment by ismailnakkar** [comment]
You're welcome! Thanks for the awesome plugin :)

**Comment by ismailnakkar** [comment]
Would you be able to relase an update please? I feel that the last version is worse than the previous one. And I really enjoy your plugin, thanks a lot

**Comment by Roboroads** [comment]
I'm sorry, time availability has not been kind these past weeks. I have some family heath matters that are taking up time at the moment. Fixing this is on top of my to-do list for when I have time.

**Comment by lucca257** [comment]
some feedback about this thread? I use too much this plugin but I use too much dd to debug short functions and other fings

**Comment by Roboroads** [comment]
![image](https://user-images.githubusercontent.com/4398301/223384044-3d675691-ced8-4d86-aef7-e533ee10f841.png)
I posted the following question to the JetBrains slack, this is the problem I'm facing to fixing this issue.

**Comment by Roboroads** [comment]
Good news everyone!

![image](https://user-images.githubusercontent.com/4398301/223566772-68f2cb36-631e-4b26-9e5b-81f0e8d59dbc.png)

I got DD to output with ANSI characters. This is finally fixable.

**Comment by Roboroads** [comment]
So, I got it working for *local* interpreters now. Things I have to tackle now:

- [ ] Remote / Docker interpreters
- [ ] dd() does not show colors (dd outputs with Symfony Vardumper and tinker outputs with Symfony Console)

**Comment by SaintPeter** [comment]
@Roboroads Installed the beta, I get the following error when doing a `dump` command:
```
Fatal error: Uncaught Error: Call to undefined method Symfony\Component\Console\Terminal::setColorMode() in Command line code:11
Stack trace:
#0 {main}
  thrown in Command line code on line 11
```

Example code:
```php
dump(['Bob']);
```

**Comment by Roboroads** [comment]
@SaintPeter You're right, I left a thing in there for tryout without removing (or needing) it.
Here's beta2 without that one line :P

[Laravel Tinker-2.6.0-beta2.zip](https://github.com/Roboroads/laravel-tinker/files/10962642/Laravel.Tinker-2.6.0-beta2.zip)

**Comment by SaintPeter** [comment]
@Roboroads Ok, beta2 resolves the immediate issue. As expected, no color for DD/dump. 

I've got it installed now, so I'll let you know if I run into any other issues.

**Comment by Roboroads** [comment]
Issue is fixed in 2.6.0 - expect update somewhere starting next week :)
